packages/ckeditor5-table/tests/tablecellproperties/tablecellpropertiesediting.js

```diff
@@ -46,7 +46,7 @@ describe( 'table cell properties', () => {
 
 			expect( config ).to.be.an( 'object' );
 			expect( config ).to.have.property( 'defaultProperties' );
-			expect( config.defaultProperties ).to.deep.equal( {} );
+			expect( config.defaultProperties ).to.include( {} );
```

Comment:
Why is this changed? What is the difference?

Reply:
oops, I reverted some code in this area and forgot to revert this comparison.